src/sql/workbench/contrib/dataExplorer/browser/extensions.contribution.ts

```diff
@@ -111,3 +111,15 @@ MenuRegistry.appendMenuItem(MenuId.DataExplorerContext, {
 		MssqlNodeContext.IsWindows, MssqlNodeContext.EngineEdition.notEqualsTo(DatabaseEngineEdition.SqlOnDemand.toString()),
 		ContextKeyRegexExpr.create('nodeType', /^(Database|Table|Column|Index|Statistic|View|ServerLevelLogin|ServerLevelServerRole|ServerLevelCredential|ServerLevelServerAudit|ServerLevelServerAuditSpecification|StoredProcedure|ScalarValuedFunction|TableValuedFunction|AggregateFunction|Synonym|Assembly|UserDefinedDataType|UserDefinedType|UserDefinedTableType|Sequence|User|DatabaseRole|ApplicationRole|Schema|SecurityPolicy|ServerLevelLinkedServer)$/))
 });
+
+// Import Database
+MenuRegistry.appendMenuItem(MenuId.DataExplorerContext, {
+	group: 'export',
+	order: 8,
+	command: {
+		id: IMPORT_DATABASE_COMMAND_ID,
+		title: localize('importDatabase', "Import New Database Project")
+	},
+	when: ContextKeyExpr.and(MssqlNodeContext.NodeProvider.isEqualTo(mssqlProviderName),
+		MssqlNodeContext.NodeType.isEqualTo(NodeType.Database), MssqlNodeContext.EngineEdition.notEqualsTo(DatabaseEngineEdition.SqlOnDemand.toString()))
```

Comment:
Move this up so that the commands actually appear in the same order as their order number